### ISSUE TITLE
If you choose plain CSS - no CSS gets included

### ISSUE THREAD

**Issue by nigel-dewar** [Swiip/generator-gulp-angular#617]
When you choose plain css, no css gets included. So when you load gulp project for first time, it looks like a page from 1995. So you have to select SASS, or LESS or something. but plain CSS gets you nothing

**Comment by zckrs** [comment]
This issue is relative to https://github.com/twbs/bootstrap/blob/master/bower.json#L18 

They don't define a main proprety for plain CSS cause wiredep can't retrieve the right file. 
If you replace this line by `"dist/css/bootstrap.css"` all right fine.

**Comment by zckrs** [comment]
You should open a issue on https://github.com/twbs/bootstrap/issues

But you can use `overrides` like ugly walkthrough in your `bower.json`
```
...
  "overrides": {
    "bootstrap": {
      "main": "dist/css/bootstrap.css"
    }
  }
...
```

**Comment by nigel-dewar** [comment]
Hi There,

 

Replace it with what?

**Comment by zckrs** [comment]
See my previous post

**Comment by nigel-dewar** [comment]
Ok, cool,

 

I see what you mean.

 

Thank you :)

**Comment by Swiip** [comment]
@zckrs your workaround is great, I agree bootstrap should fix their `bower.json`. Still, I think we can't accept to deliver broken projects to our users because one of our dependency is bugged. We should add the override option in the generator as long as bootstrap doesn't fix the problem.

**Comment by nigel-dewar** [comment]
I agree.

**Comment by zckrs** [comment]
Fixed by [v0.12.1](https://github.com/Swiip/generator-gulp-angular/releases/tag/v0.12.1)
